crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -1543,9 +1592,19 @@ impl CumulocityConverter {
                 return Ok(vec![]);
             }
         };
-        let ops_file = ops_dir.join(c8y_operation_name);
+
         create_directory_with_defaults(&*ops_dir)?;
-        create_file_with_defaults(ops_file, None)?;
+        if let Some((template_name, Some(operation_name))) = self
+            .operations
+            .match_template_with_operation_by_command_name(c8y_operation_name)
+        {
+            let template_file = self.config.ops_dir.join(&template_name);
+            let ops_file = ops_dir.join(&operation_name);
```

Comment:
This `operation_name` is equal to `c8y_operation_name`.

```suggestion
            let ops_file = ops_dir.join(&c8y_operation_name);
```

It means we can simplfy `match_template_with_operation_by_command_name()`. It can return only single value.